### DISCUSSION TITLE
Roadmap for 5G SA Core?

### DISCUSSION THREAD

**Discussion by luiborge** [magma/magma#8474] [Q&A]
Hello,

Do we have a roadmap for 5G SA Core?
Do we have a list of features currently supported by Magma?

Best Regards
Luis

**Comment by pkapatsos-blinq** [comment]
Check here https://app.zenhub.com/workspaces/magma-5fac75d3e2cd890011f1677a/roadmap

**Reply by vikram-wani** [reply]
thank you @pkapatsos-blinq this is a good resource. However if I am looking to find feature planned for 1.7 and 1.8, is there a roadmap available for that?
